src/db/migrations/20230907143204_add_onerep_scanresults_table.js

```diff
@@ -100,6 +104,8 @@ export async function down(knex) {
   });
   await knex.schema.dropTable("onerep_scan_results");
   await knex.schema.alterTable("onerep_scans", (table) => {
+    // @ts-ignore TODO: Determine if the following line should be changed to
+    // `defaultTo` or remain unchanged as initially deployed.
```

Comment:
FYI this is `dropUnique` not `default`, which should be valid https://knexjs.org/guide/schema-builder.html#dropunique